src/basic/mountpoint-util.c

```diff
@@ -22,6 +22,7 @@
 #include "nulstr-util.h"
 #include "parse-util.h"
 #include "path-util.h"
+#include "process-util.h"
```

Comment:
Any reason this was added? The new helper is in fd-util.h

Reply:
For getpid_cached()

Reply:
ah, missed that